docs/deploy-development-rp.md

```diff
@@ -33,17 +35,17 @@
    cd ${GOPATH:-$HOME/go}/src/github.com/Azure/ARO-RP
    ```
 
-1. If you don't have access to a shared development environment and secrets,
-   follow [prepare a shared RP development
-   environment](prepare-a-shared-rp-development-environment.md).
-
 1. Set SECRET_SA_ACCOUNT_NAME to the name of the storage account containing your
    shared development environment secrets and save them in `secrets`:
 
    ```bash
    SECRET_SA_ACCOUNT_NAME=rharosecrets make secrets
    ```
 
+   If you don't have access to a shared development environment and secrets,
+   follow [prepare a shared RP development
+   environment](prepare-a-shared-rp-development-environment.md).
+
```

Comment:
revert - the order was correct before for the case that a new SDE has to be created

Reply:
The reason I changed this order is because a) it confused me b) I feel like the audience for this should be the most common case.

If someone is making a new shared dev environment, they know that this isn't set up yet. But someone setting this up for the first time doesn't. But I'm not attached to this, I can revert.